pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/RouteTest.java

```diff
@@ -1239,8 +1244,10 @@ private void addParticipant(String json, int instanceId, long maxParticipantId,
                 try {
                     inTransaction((conn) -> {
                         try {
-                            ddpMedicalRecordDataRequest.writeInstitutionBundle(conn, instanceId, participantInstitution,
-                                    instanceName);
+                            DDPInstanceDto instanceDto = new DDPInstanceDto.Builder()
```

Comment:
I needed to make this change, but also fixed the checkstyle violated since they are annoying.